app/webpacker/components/RegistrationsV2/Register/index.jsx

```diff
@@ -53,22 +58,39 @@ function Register({
     },
   });
 
+  const onTimerEnd = useCallback(() => {
+    setTimerEnded(true);
+  }, [setTimerEnded]);
+
+  if (isFetching) {
+    return <Loading />;
+  }
+
+  if (userCanPreRegister(userInfo, competitionInfo) || competitionInfo['registration_currently_open?'] || timerEnded) {
+    return (
+      isFetching ? <Loading />
```

Comment:
The is-fetching case is handled above now, right? Doesn't need to be here.

Reply:
copy paste whoopsie